newsrevenuehub/rev-engine
# #11349 add required static disclaimer text

#### What's this PR do?
Implements [this basecamp todo item](https://3.basecamp.com/3120230/buckets/21497549/todos/4500174537). Specifically:
Adds some different text below the submit button on donation pages. This text will remain static even after we introduce a "new page template" that includes some small-print near this section.

#### How should this be manually tested?
View a donation page and observe the text. Notice how it changes when you change the amount, frequency, or payFees toggle.

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No
